warehouse/manage/views.py

```diff
@@ -339,16 +337,7 @@ def _key(project):
     effective_principals=Authenticated,
 )
 def manage_project_settings(project, request):
-    release_count = (
-        request.db.query(Release)
-        .filter(Release.project == project)
-        .count()
-    )
```

Comment:
Still might want to do this eventually, it's kinda wasteful to load all the releases into memory just to count them.